src/math/operations/cutPasteOps/augment.ts

```diff
@@ -313,12 +313,19 @@ const hasGyrateOpts = (polyhedron: Polyhedron) => {
   return false;
 };
 
-export const augment = makeOperation('augment', {
+type GyrateOpts = 'ortho' | 'gyro';
+
+interface Options {
+  face: Face;
+  gyrate?: GyrateOpts;
+  using?: string;
+}
+export const augment = makeOperation<Options>('augment', {
   apply(polyhedron, { face, gyrate, using }) {
     const augmentType = using
       ? augmentTypes[using[0]]
-      : defaultAugmentType(face.numSides);
-    return doAugment(polyhedron, face, augmentType, gyrate);
+      : defaultAugmentType(face!.numSides);
```

Comment:
ugh, we shouldn't need to do this. I don't think `apply` needs `Partial<Options>`, we just need to fix it...